tools/clang/test/DXC/denorm.test

```diff
@@ -0,0 +1,9 @@
+// Test for denorm options
+// RUN: %dxc  %S/Inputs/smoke.hlsl /Tps_6_2 /denorm preserve | FileCheck %s
+// CHECK:"fp32-denorm-mode"="preserve"
+
+// RUN: %dxc  %S/Inputs/smoke.hlsl /Tps_6_2 /denorm ftz | FileCheck %s --check-prefix=ftz
+// CHECK:"fp32-denorm-mode"="ftz"
+
+// RUN: not %dxc  %S/Inputs/smoke.hlsl /Tps_6_2 /denorm abc
```

Comment:
Should we check for errors here?

Should we add a run line to verify that the `fp32-denorm-mode` metadata doesn't appear if you don't pass the flag?